remarkable/RemarkableWindow.py

```diff
@@ -1068,29 +1068,48 @@ def on_menuitem_bullet_list_activate(self, widget):
             start_iter = self.text_buffer.get_iter_at_line(line_number)
             self.text_buffer.insert(start_iter, "- ")
 
+    def _text_buffer_get_iter_at_line_end(self, line_number):
```

Comment:
Maybe there is a better way of getting the end of line iterator, if so feel free to change this